lineapy/plugins/jinja_templates/module_test.jinja

```diff
@@ -0,0 +1,36 @@
+import unittest
+import warnings
+from typing import Any, Dict
+
+from {{ MODULE_NAME }} import (
+{%- for func_data in FUNCTION_METADATA_LIST %}
+    {{ func_data.function_name }},
+{%- endfor %}
+)
+
+class {{ TEST_CLASS_NAME }}(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls) -> None:
+        # Adjust as needed
```

Comment:
would like more comments here for the user's guide. eg this will run before all of the tests.
maybe create another setUp skeleton method so that users can have hooks to do whatever they want before every test.

Reply:
Done